dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+# Created by Barbara Ellynes Zucchi Nobre Silva for the newsmap package: https://github.com/koheiw/newsmap
+
+AFRICA:
+  EAST:
+    'BI': [Burundi, Burúndi, Burundian*, Burundiense*, Burundinês*, Bujumbura]
+    'DJ': [Djibouti, Jibuti, Djiboutian*]
+    'ER': [Eritreia, Eritreu*, Eritréia*, Eritreense*, Asmara]
+    'ET': [Etiópia, Etíope*, Etiopian*, Adis Abeba]
+    'KE': [Quénia, Quênia, Quenian*, Nairóbi]
+    'KM': [Comores, Comoriense*, Comorian*, Moroni]
+    'MG': [Madagáscar, Madagascar, Madagascarense*, Malgaxe*, Antananarivo]
+    'MU': [Maurícia, Maurício, Maurician*, Porto Luís]
+    'MW': [Malawi, Maláui, Malaui, Malauian*, Malawian*, Malavian*, Malawiense*, Malauiense*, Malavita*, Lilongué, Lilôngue, Lilóngue, Lilongue, Lilongwe]
+    'MZ': [Moçambique, Moçambican*, Maputo]
+    'RE': [Reunião, Ilha da Reunião, Reunionense*, Reunionês*, Reuniones*]
```

Comment:
Reunião would lead to false matches, as it also means "meeting", so I wonder whether maybe just use "Ilha da Reunião". Or, if keeping Reunião, then "Ilha da Reunião" wouldn't be needed.

Reply:
The reason why I had added both was because "Reunião" shows as the most known name for the Island when I searched, but you are right that could lead to too many confusions (I was unsure about what to do). So I will just keep the official name in the dictionary.